examples/ui-demo/src/components/configuration/Configuration.tsx

```diff
@@ -0,0 +1,53 @@
+// import { useState } from "react";
+import { cn } from "@/lib/utils";
+
+import { SettingsIcon } from "../icons/settings";
+// import { HelpTooltip } from "../shared/HelpTooltip";
+import { WalletTypeSwitch } from "../shared/WalletTypeSwitch";
+import ExternalLink from "../shared/ExternalLink";
+import { useConfigStore } from "@/state";
+import { WalletTypes } from "@/app/config";
+
+export const Configuration = ({ className }: { className?: string }) => {
+  const { setWalletType, walletType } = useConfigStore();
+  // const [walletType, setWalletType] = useState(WalletTypes.smart);
+
+  const onSwitchWalletType = () => {
+    setWalletType(
+      walletType === WalletTypes.smart
+        ? WalletTypes.hybrid7702
+        : WalletTypes.smart
+    );
+  };
+
+  return (
+    <div className={cn("flex flex-col", className)}>
+      <div className="flex flex-row gap-2 items-center pb-5">
+        <SettingsIcon />
+        <span className="font-semibold">Configuration</span>
+      </div>
+      <div className="flex flex-row items-center gap-2 pb-2">
+        <h4 className="font-normal text-sm text-secondary-foreground">
+          Embedded Wallet Type
+        </h4>
+        {/* <HelpTooltip text="An account powered by a smart contract to enable more features. Not an EOA. Recommended for new wallets." /> */}
+      </div>
+      <WalletTypeSwitch
+        id="theme-switch"
```

Comment:
❗ i think this was copied from the ThemeSwitch and would result in 2 elements on the page w/ the same id